### DISCUSSION TITLE
Dai stuck at wormhole - from Ethereum to Solana

### DISCUSSION THREAD

**Discussion by CryptoVincenzo** [wormhole-foundation/wormhole#254] [Problems / bugs] [answered]
I was using wormhole bridge to transfer ERC20 Dai to Solana. I tried this process twice. The 1st time I did it with 100 Dai, and it got stuck at “Awaiting final confirmation on Solana”. And then I tried for the second time with 50 Dai and the process was completed and I was able to see 50 wDai in my Solana wallet. Below is what i could gather as transaction id and my Solana wallet address, please help. What can we do to avoid this from happening in the future? Thanks team. 

Solana wallet address: jzjUkn4se5fqfb6yg8XJW5MyrQqPJaee1oMGL9dJsg2

Ethereum wallet address: 0xc5F1F580084Ccf817c4834a09709bDdC6469F5DD

Token account: https://solscan.io/account/9kdB2HHDTgfQpZ6QWqW3GUDsNqgL3ZJmRqgromzVeZBf

Transaction details (i think this is the failed transaction on the Solana side): https://solscan.io/tx/zr6RrBLz9u7LTYJdZbk1MZtYpWUvN1Hm2vity6BxABMZpiJm8QB5LqHbHJLNxVTaupw1V5gzJiR4gFcQoe36UAk

Transaction details on the Ethereum side where the 100 Dai is confirmed to be locked: 
https://etherscan.io/tx/0x6e8d7389583d84527a26f0ee63e10b600a47724d91f1f8594318562c09cf0d0d

**Comment by hendrikhofstadt** [comment, marked as answer]
Completed in https://explorer.solana.com/tx/55Kq1fZF9KP9KnsKu4RGwcfwL4kzq4AAXJpTszSeRzZj8G48rcaut3AKNozAd2xF9J2D49zJv2SYkhaHuYZFVx6f

**Reply by CryptoVincenzo** [reply]
Thanks very much Hendrik, I see the wDai in the wallet now. Looking forward to using the more stable v2 you mentioned on the other thread. Any ETA on that?
